### PR TITLE
Add install script for macOS and linux

### LINKED ISSUE #150: install script for linux/darwin
- [ ] ubuntu
- [ ] debian
- [ ] arch
- [ ] macOS (homebrew)
- [ ] centOS
- [ ] fedora
- [ ] alpine

### PR DESCRIPTION
**Pull request by nhooyr**
- [x] Update docs

Closes #150

### CODE REVIEW

**Comment by nhooyr** [comment]
Do you guys think it's worth making the script POSIX instead of bash? Would make it uglier but work across more platforms.

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	elif command -v wget > /dev/null; then
+		wget "$1" -O -
+	fi
+}
+
+latestReleaseURL() {
+	log "finding latest release"
+	local os=$1
+	download https://api.github.com/repos/cdr/sail/releases/latest |
+		jq -r ".assets[]
```
Comment (scsmithr) [inline install.sh]: I don't think it's a good idea to assume `jq` is installed.

Comment (nhooyr) [reply install.sh]: That's a fair point. It just makes it so nice and easy 😭

Comment (scsmithr) [reply install.sh]: Jank shell solution:
```
curl https://api.github.com/repos/cdr/sail/releases/latest | grep "browser_download_url" | grep "linux" | awk -F": " '{print $2}'
```

Comment (ammario) [reply install.sh]: yeah definitely can't use jq...

**Comment by ammario** [comment]
Also should amend docs

**Comment by ammario** [comment]
@nhooyr isn't bash universal these days?

**Comment by nhooyr** [comment]
Yea pretty much, I guess its nbd, lets see if anyone wants a POSIX compliant script after release.

**Pull request merged**